CMakeLists.txt

```diff
@@ -60,9 +60,15 @@ endif ()
 
 if (USE_SANITIZERS)
     # Recommended flags per https://github.com/google/sanitizers/wiki/AddressSanitizer
-    set(sanitizer_cxx_flags
-        "-fsanitize=${USE_SANITIZERS} -fno-omit-frame-pointer -fno-optimize-sibling-calls -O1 -shared-libsan"
-    )
+    string(
+        JOIN
+        " "
+        sanitizer_cxx_flags
+        "-fsanitize=${USE_SANITIZERS}"
+        "-fno-omit-frame-pointer"
+        "-fno-optimize-sibling-calls"
+        "-O1"
+        "-shared-libsan")
```

Comment:
Curious: why is this better?

Reply:
I did this to make this line shorter, but the code is arguably worse.

Since we already break long lines during formatting I'll just disable the line length lint since it just fires for isolate long strings or variable names at this point which are not too.